.github/workflows/pypi-publish.yml

```diff
@@ -10,10 +10,10 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
           python-version: 3.7
```

Comment:
We can probably bump to 3.9: 3.7 is not supported anymore and 3.9 is the version required by most supported Invenio applications.